l10n/templates/strings.properties

```diff
@@ -181,3 +186,24 @@ manual_migration_cancel_button=No Thanks
 # LOCALIZATION NOTE (manual_migration_import_button): This message is shown on a button that starts the process
 # of importing another browser’s profile profile into Firefox.
 manual_migration_import_button=Import Now
+
+# LOCALIZATION NOTE (error_fallback_default_info).  This message is shown when
+# there is an error rendering some part of the page in place the chunk of UI
```

Comment:
English seems off in this comment (or, at least, I can't understand it at all)

Reply:
@dmose wrote the note but is out.. so I'll take a stab ;)

```ini
# LOCALIZATION NOTE (error_fallback_default_*): This message and suggested
# action link are shown in each section of UI that fails to render
error_fallback_default_info=Oops, something went wrong loading this content. 
error_fallback_default_refresh_suggestion=Refresh page to try again.
```

Reply:
@dmose, @flodolo does this NOTE adjustment seem reasonable?

Reply:
Makes definitely sense to me